spec/factories/service_providers.rb

```diff
@@ -33,6 +34,16 @@
       end
     end
 
+    trait :idv do
+      ial { 2 }
+    end
+
+    # Can we enable this globally?
+    trait :active do
+      active { true }
+    end
```

Comment:
I want to verify with tests that it's OK to put this in the base `service_provider` factory.